src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -215,6 +215,10 @@ public void setPositive(Boolean positive) {
         this.positive = positive;
     }
 
+    public void setPositiveViaCredits() {
+        this.positive = credits >= 0;
```

Comment:
```suggestion
        this.positive = credits != null && credits >= 0;
```
From the comment above the other change in this file, I assume that credits can actually be `null`.